shared/.gitignore

```diff
@@ -0,0 +1,2 @@
+../browser/keybase-chrome-1.10.14.zip
+../browser/keybase-firefox-1.10.14.zip
```

Comment:
I think these lines should be in our root .gitignore file instead of a separate one in shared/. Someone else could probably say better than I could, though.

Reply:
oops.